website/docs/r/team.html.markdown

```diff
@@ -58,6 +58,7 @@ The `organization_access` block supports:
 * `manage_teams` - (Optional) Allow members to create, update, and delete teams.
 * `manage_organization_access` - (Optional) Allow members to update the organization access settings of teams.
 * `access_secret_teams` - (Optional) Allow members access to secret teams up to the level of permissions granted by their team permissions setting.
+* `manage_secret_teams` - (Optional) Allows members to create, edit, and delete agent pools within their organization.
```

Comment:
Suggestion: I believe that `manage_agent_pools` is what we're looking for here 😊 :
```suggestion
* `manage_agent_pools` - (Optional) Allows members to create, edit, and delete agent pools within their organization.
```

Reply:
omg